app/src/main/java/com/github/dedis/student20_pop/model/Event.java

```diff
@@ -0,0 +1,136 @@
+package com.github.dedis.student20_pop.model;
+
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Class modeling an Event
+ */
+public class Event {
+
+    private String name;
+    private long time;
+    private String id;
+    private String lao;
+    private List<String> attendees;
+    // Can use GeoLocation in the future
+    private String location;
+    // Can use enums in the future
+    private String type;
+    private JSONObject other;
```

Comment:
What is that used for ?

Reply:
It appears in the Protocol Messaging Definition, not sure of the usage yet.